### DISCUSSION TITLE
What is the difference between gpaste and gpaste-2 packages on Ubuntu?

### DISCUSSION THREAD

**Discussion by t1u1** [Keruspe/GPaste#448] [Q&A]
After upgrading from Ubuntu 22.04 to 24.04, the package gpaste has been uninstalled automatically by the upgrade process, and is no longer available in the repository.

However a new package called gpaste-2 is now available. Can someone tell what is the reason behind this? Searching on the web didn't bring up anything.

**Comment by Keruspe** [comment]
There were breaking under-the-hood changes and Ubuntu flagged them in the packages name, but it's just a "regular" update in the end for nearly everyone
